prediction_market_agent_tooling/tools/rephrase.py

```
@@ -0,0 +1,71 @@
+    needed_parent_outcome: str,
+    engine: str = "gpt-4.1",
+    temperature: float = LLM_SUPER_LOW_TEMPERATURE,
+    seed: int = LLM_SEED,
+    prompt_template: str = REPHRASE_QUESTION_PROMPT,
+    max_tokens: int = 1024,
+) -> str:
+    try:
+        from langchain.prompts import ChatPromptTemplate
+        from langchain_openai import ChatOpenAI
+    except ImportError:
+        raise ImportError("langchain not installed")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)